README.adoc

```diff
@@ -10,6 +10,18 @@
 
 Want to help? We love to hear that! Below are a few hints to guide you.
 
+1. Create a new recipe file under recipes/<appropriate_directory>/
+2. Make sure you include a section identifier `[id='sec.recipe_name']` and a title `=== Recipe Name`
+   each on their own line and separated by an empty line from other content
+3. Add a reference to the new recipe to recipes.adoc in the top level
+4. Add any images to images/<recipe_name>/
+5. build the docs with docbuild.sh
+6. check the output at build/recipes.html to verify it matches the formatting you want
+
+It is recommended to look at some of the existing recipes for copy the formatting for
```

Comment:
```suggestion
It is recommended to look at the existing recipes to copy the formatting for
```

Reply:
I've added a recipe_template.adoc, probably good to reference this too.

Reply:
thanks for doing that! great idea! I added a couple references to it